File: c/crescente.c
#include <stdio.h>

int main()
{
    int x, y;
    printf("Digite dois numeros:\n");
    scanf("%d", &x);
    scanf("%d", &y);

    if (x < y) {
        printf("CRESCENTE!\n");
    }
    else if (x > y){
        printf("DECRESCENTE!\n");
    }

    while (x != y){
        printf("Digite outros dois numeros:\n");
        scanf("%d", &x);
        scanf("%d", &y);

        if (x < y) {
        printf("CRESCENTE!\n");
    }
    else if (x > y){
        printf("DECRESCENTE!\n");
    }

    }

    return;
}

File: python/crescente.py
print("Digite dois numeros:")
x = int(input())
y = int(input())

if x < y:
    print("CRESCENTE")
elif y < x:
    print("DECRESCENTE")

while x != y:
    print("Digite outros dois numeros:")
    x = int(input())
    y = int(input())

    if x < y:
        print("CRESCENTE")
    elif x < y:
        print("DECRESCENTE")

